<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSignatureDialog</class>
 <widget class="QDialog" name="TimeSignatureDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Signature</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="showTimeSignatureLabel">
       <property name="text">
        <string>Show Time Signature:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="showTimeSignature"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="commonTimeLabel">
       <property name="text">
        <string>Common Time:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="commonTime"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cutTimeLabel">
       <property name="text">
        <string>Cut Time:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="cutTime"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="beatsPerMeasureLabel">
       <property name="text">
        <string>Beats Per Measure:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="beatsPerMeasure">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="beatValueLabel">
       <property name="text">
        <string>Beat Value:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="beatValue">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="metronomePulsesPerMeasureLabel">
       <property name="text">
        <string>Metronome Pulses Per Measure:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="metronomePulses">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="beamingPatternLabel">
       <property name="text">
        <string>Beaming Pattern:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="beamingPatternsGroup">
       <item>
        <widget class="QLineEdit" name="beamingPattern1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="beamingPattern2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="beamingPattern3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="beamingPattern4"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TimeSignatureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TimeSignatureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
